lib/index.js

```diff
@@ -126,6 +155,7 @@ Ramen.prototype.identify = function(identify){
 
   // For tracking purposes
   ramenSettings._partner = 'segment.com';
+  ramenSettings.disable_location_watch = true;
```

Comment:
Ahh ok yeah good catch @ndhoule 

Ramen doesn't care about anything until `#identify` is called. Each subsequent call to `#page`, `#group` and `#identify` effectively reloads the entire RamenJS environment.

So I think what you're saying is that I should put checks in place on `Ramen.prototype.(page|track|group)` that make them a no-op until `#identify` has been called. That right?

As I re-read this, it hits me: are Segment users instructed to call `#group` only after `#identify` or is there chance some of your customers are calling `#group` before `#identify` all the time and expecting the partner libraries to save the data from `#group` and apply it after the `#identify`?